tests/lib/cmsis_dsp/matrix/Kconfig

```diff
@@ -12,7 +12,6 @@ config CMSIS_DSP_TEST_MATRIX_UNARY_Q31
 
 config CMSIS_DSP_TEST_MATRIX_UNARY_F16
 	bool "Test: Matrix Unary F16"
-	depends on CMSIS_DSP_FLOAT16
```

Comment:
should not remove this dependency since the F16 implementation is not available unless `CMSIS_DSP_FLOAT16` is enabled.